XRFoundation/XREngine
# Network xr hands

## Summary

Networking of XR hands


## Checklist
- [x] CI/CD checks pass `npm run check`
  - [x] Linter passing via `npm run lint`
  - [x] Typescript passing via `npm run check-errors`
  - [x] Unit & Integration tests passing via `npm run test`
  - [ ] Docker build process passing via `npm run build-client`
- [ ] Changes reviewed by at least 2 approved reviewers


## QA Steps
Open 2 instances of a test location.
Set avatar controller type to XR hands from the menu.
Launch VR mode. 
XR Hands movements should be synced across all sessions.


## Reviewers
@HexaField
